CMake/Toolchain-RaspberryPi.cmake

```diff
@@ -0,0 +1,19 @@
+SET(CMAKE_SYSTEM_NAME Linux)
+SET(CMAKE_SYSTEM_VERSION 1)
+
+# Specify the cross compiler
+SET(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
+SET(CMAKE_CXX_COMPILER arm-linux-gnueabihf-g++)
+
+# Where is the target environment
+SET(CMAKE_FIND_ROOT_PATH $ENV{HOME}/rpi/rootfs)
```

Comment:
This assumes we already have such a directory. Either something in README on how to set up the project would help or some automated script that can get that for us.

Also, this looks like a nice thing to turn into variable as some may have it in ~/rpi, others in ~/projects, yet others in /var/lib/rpi